src/pages/Product/Product.js

```diff
@@ -1,8 +1,48 @@
 import React, { Component } from 'react';
 
+import ProductCard from './ProductCard/ProductCard';
+
+import './Product.scss';
+
 export class Product extends Component {
+  constructor() {
+    super();
+    this.state = {
+      productCard: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('/data/ProductData.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          productCard: data,
+        });
+      });
+  }
+
   render() {
-    return <div>Product</div>;
+    return (
+      <div className="Product">
+        <header className="ProductHeader">
+          <h1>
+            건강한 삶을 위한
+            <br />
+            빌리의 연구와 도전은 계속됩니다.
+          </h1>
+        </header>
+        <section className="ProductBody">
+          {/* <h2 className="sr-only">Product Body</h2> */}
+          {/* 맵함수의 고유한 값으로 symbol 가능? */}
+          <ul className="ProductList">
+            {this.state.productCard.map(product => {
+              return <ProductCard key={product.id} productCard={product} />;
+            })}
```

Comment:
```suggestion
            {this.state.productCard.map(product => <ProductCard key={product.id} productCard={product} />)}
```
arrow function 에서 바로 리턴만 할 경우에는 위와 같이 증괄호랑 return 키워드 생략할 수 있습니다!

Reply:
- 중괄호 return ; 생략하여 코드 리팩토링 하였습니다 !